tests/test_corporate.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import inspect
+
+from drhtaxes import corporate
+from drhtaxes.corporate import COR
+
+
+class ApplicationTests(unittest.TestCase):
+    """
+    Testing class_COR
+    """
+    
+    def test_class_COR(self):
+        """
+        Testing function_corp
+        """
+        tax = COR()
+        print(tax.corp(79000, 0))
+        self.assertEqual(15800, tax.corp(79000, 0))
+        #self.assertTrue(tax.corp(79000,0) == 15800)
+
```

Comment:
Yes we haven't covered a range of possible inputs yet, what happens if I input 30000, 2000 or 1000, 3000